tools/bundle-size-tools/README.md

```diff
@@ -113,7 +113,7 @@ If the artifacts are not available for the baseline commit, the buddy build adds
 
 ## Build runs main CI completes to update pull requests pending baseline stats
 
-There is a separate build pipeline that runs after each successful main build and looks for all buddy builds with the `bundle-buddy-pending-<commitHash>` corresponding to the `<commitHash>` of the main build. This build process will then run Bundle buddy on all these PRs and post a comment.
+There is a separate build pipeline that runs after each successful main build and looks for all buddy builds with the `bundle-buddy-pending-<commitHash>` corresponding to the `<commitHash>` of the main build. This build process will then run bundle-size-tools on all these PRs and post a comment.
```

Comment:
Has the build tag been updated with the new name? Or is it still bundle-buddy-pending-<commitHash>? If it is, you should note that the command is correct; otherwise I would just assume it's a typo. :)

Reply:
was correct because i didn't change the tag yet but i'll change it now